sbi/utils/torchutils.py

```diff
@@ -208,7 +208,9 @@ def __init__(
 
 
 # XXX does an in-place version (e.g. make_conform_) make sense?
-def make_shapes_conform(parameter: torch.Tensor, observation: torch.Tensor) -> (torch.Tensor, torch.Tensor):
+def add_batch_dim(
+    parameter: torch.Tensor, observation: torch.Tensor
+) -> (torch.Tensor, torch.Tensor):
     """
     Return tensors that both have the same tensor.ndim
     Function now also covers cases where parameters is ndim=1 and observation ndim=2
```

Comment:
'now' is not defined when looking at this function in 2023. This kind of reference is good for a commit message, not for a docstring.